SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Supported Versions
+
+To be confirmed.
+
+## Reporting a Vulnerability
+
+To report a security problem in Kanister, please contact the maintainers listed
+in the [MAINTAINERS.md](MAINTAINERS.md) file.
```

Comment:
TODO - Setup google workspace for kanister so we can have a

security@kanister.io mailing list